src/Docker/Runner/DataLoader/DataLoader.php

```diff
@@ -313,7 +313,11 @@ private function validateStagingSetting()
             (($this->getStagingStorageInput() === Reader::STAGING_SNOWFLAKE) &&
                 ($this->getStagingStorageOutput() !== Reader::STAGING_SNOWFLAKE)) ||
             (($this->getStagingStorageOutput() === Reader::STAGING_SNOWFLAKE) &&
-                ($this->getStagingStorageInput() !== Reader::STAGING_SNOWFLAKE))
+                ($this->getStagingStorageInput() !== Reader::STAGING_SNOWFLAKE)) ||
+            (($this->getStagingStorageInput() === 'workspace-synapse') &&
+                ($this->getStagingStorageOutput() !== 'workspace-synapse')) ||
+            (($this->getStagingStorageOutput() === 'workspace-synapse') &&
+                ($this->getStagingStorageInput() !== 'workspace-synapse'))
```

Comment:
Toto pak bude tiez z Input Mappingu `Reader::STAGING_SYNAPSE` ?

Reply:
jojo, v podstate se to melo pridat az s updatem IM, ale zase bych to musel vyhodit z testu, tak jsem to dal zatim takhle a pak to predelam na konstanty